### ISSUE TITLE
Dataset request example: aC_JCP2023 

### ISSUE THREAD

**Issue by jvita** [colabfit/data-lake#16]
### Name

Josh Vita

### Email

vita1@llnl.gov

### Dataset name

aC_JCP2023 

### Authors

Emi Minamitani, Ippei Obayashi, Koji Shimizu, Satoshi Watanabe

### Links

* Publication link: https://doi.org/10.1063/5.0159349
* Dataset link: https://zenodo.org/record/7905585

### Dataset description

The amorphous carbon dataset was generated using ab initio calculations with VASP software. We utilized the LDA exchange-correlation functional and the PAW potential for carbon. Melt-quench simulations were performed to create amorphous and liquid-state structures. A simple cubic lattice of 216 carbon atoms was chosen as the initial state. Simulations were conducted at densities of 1.5, 1.7, 2.0, 2.2, 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 to produce a variety of structures. The NVT ensemble was employed for all melt-quench simulations, and the density was adjusted by modifying the size of the simulation cell. A time step of 1 fs was used for the simulations. For all densities, only the Γ points were sampled in the k-space. To increase structural diversity, six independent simulations were performed.

In the melt-quench simulations, the temperature was raised from 300 K to 9000 K over 2 ps to melt carbon. Equilibrium molecular dynamics (MD) was conducted at 9000 K for 3 ps to create a liquid state, followed by a decrease in temperature to 5000 K over 2 ps, with the system equilibrating at that temperature for 2 ps. Finally, the temperature was lowered from 5000 K to 300 K over 2 ps to generate an amorphous structure.

During the melt-quench simulation, 30 snapshots were taken from the equilibrium MD trajectory at 9000 K, 100 from the cooling process between 9000 and 5000 K, 25 from the equilibrium MD trajectory at 5000 K, and 100 from the cooling process between 5000 and 300 K. This yielded a total of 16,830 data points.

Data for diamond structures containing 216 atoms at densities of 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3 were also prepared. Further data on the diamond structure were obtained from 80 snapshots taken from the 2 ps equilibrium MD trajectory at 300 K, resulting in 560 data points.

To validate predictions for larger structures, we generated data for 512-atom systems using the same procedure as for the 216-atom systems. A single simulation was conducted for each density. The number of data points was 2,805 for amorphous and liquid states

### File details

File format: Extended XYZ
Approximate size: 258.5 MB

Folder structure:

*  216atom_amorphous
    * batch1.xyz
    * batch2.xyz
    * batch3.xyz
    * batch4.xyz
    * batch5.xyz
    * batch6.xyz
* 216atom_crystal
    * crystal_highden.xyz
* 512atom_amorphous
    * amo512.xyz
* dataset_train_test_split
    * test.xyz
    * train.xyz

Descriptions:
* `216atom_amorphous`: Contains six xyz files generated from the trajectory of the melt-quench simulation.

* `216atom_crystal`: Contains a single xyz file with data of diamond structures containing 216 atoms at densities of 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3.

* `512atom_amorphous`: Contains a single xyz file with data of 512-atom systems.

* `dataset_train_test_split`: The training and test data used in the manuscript, constructed from splitting the entire 216atom_amorphous dataset.

### Method

DFT

### Software

VASP

### Additional details

 LDA XC-functional, PAW potential

### Property types

- [X] Atomic forces
- [ ] Atomization energy
- [ ] Band gap
- [ ] Cauchy stress
- [ ] Formation energy
- [X] Free energy
- [X] Potential energy
- [ ] Other/additional

### Property details

* potential-energy; units=eV; per_atom=false
* atomic-forces; units=eV/Ang
* free-energy; units=eV; per_atom=false

### Elements

C

### Number of Configurations

20,195

### Naming convention

Names can be generated by assigning indices to the configurations, prepended with their full path. For example: `216atom_amorphous/batch1/0, 216atom_amorphous/batch1/1, 216atom_amorphous/batch1/....`

### Configuration sets

* name; 216atom_crystal/*; Diamond structures containing 216 atoms at densities of 2.4, 2.6, 2.8, 3.0, 3.2, 3.4, and 3.5 g/cm3.
* name; 216atom_amorphous/*; Trajectories from melt-quench simulations for configurations with 216 atoms
* name; 512atom_amorphous/*; Trajectories from melt-quench simulations for configurations with 512 atoms

### Configuration labels

* process; crystal; diamond, crystal
* process; quench1; amorphous
* process; quench2; amorphous
* process; buffer_low; amorphous
* process; buffer_high; amorphous

### Distribution license

Creative Commons Attribution 4.0 International

### Permissions

- [ ] I confirm that I have the necessary permissions to submit this dataset

**Comment by gpwolfe** [comment]
staged for ingestion following next database update

**Comment by gpwolfe** [comment]
new database now live
